.github/workflows/ci.yml

```diff
@@ -402,6 +402,41 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y criu
 
+      - name: Build and install erofs-utils from source
+        if: matrix.os != 'ubuntu-24.04-arm'
+        run: |
+          # Install dependencies
+          sudo apt-get update
+          sudo apt-get install -y wget make gcc libz-dev liblz4-dev libfuse-dev autoconf libtool pkg-config uuid-dev
+
+          # Create temporary build directory
+          BUILD_DIR="$(mktemp -d)"
+          cd "$BUILD_DIR"
+
+          # get erofs-utils-1.8.8.tar.gz
+          wget https://git.kernel.org/pub/scm/linux/kernel/git/xiang/erofs-utils.git/snapshot/erofs-utils-1.8.8.tar.gz
+          tar -xzf erofs-utils-1.8.8.tar.gz
+          cd erofs-utils-1.8.8
+
+          # Build and install erofs-utils
+          ./autogen.sh
+          ./configure --enable-lz4 --enable-fuse --with-uuid
+          make -j"$(nproc)"
+          sudo make install
+          sudo ldconfig
+
+          # Clean up
+          cd -
+          rm -rf "$BUILD_DIR"
+
+          # Verify installation
+          mkfs.erofs --version
+
+      - name: Load EROFS kernel module
+        if: matrix.os != 'ubuntu-24.04-arm'
```

Comment:
we do we need to skip 24.04 arm?

Reply:
not quite sure what happened on 24.04 arm, all ubuntu-azure should have erofs ko now. Cc @aadhar-agarwal

Reply:
Had tested previously with ARM and the erofs kernel module was not being loaded. However, I removed the if condition and it seems to be loading now. Thanks!